validation/charts/appco/istio.go

```diff
@@ -0,0 +1,78 @@
+package appco
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	extencharts "github.com/rancher/shepherd/extensions/charts"
+	"github.com/rancher/shepherd/extensions/kubectl"
+	namegen "github.com/rancher/shepherd/pkg/namegenerator"
+	"github.com/rancher/tests/actions/charts"
+	kubeapiNamespaces "github.com/rancher/tests/actions/kubeapi/namespaces"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	RancherIstioSecret string = "application-collection"
+)
+
+func createIstioNamespace(client *rancher.Client, clusterID string) error {
```

Comment:
imo we shouldn't write helpers for a one liner, this would be pretty simple to just have inline.